clab/inventory.go

```diff
@@ -243,6 +243,7 @@ func (c *CLab) generateNornirSimpleInventory(w io.Writer) error {
 			}
 
 		}
+		nornirNode.NornirGroup = n.Config().Labels["nornir-group"]
```

Comment:
@mpfz0r you can probably allow setting multiple groups with comma separated values

Reply:
Yes that's a possibility, albeit a hackish one.
I wanted to see if a nicer way is possible. But I can also implement it this way...

Reply:
another way would be to collect all labels that start with `nornir-group`
eg.
```
labels:
  nornir-group-1: foo
  nornir-group-2: bar
```

Reply:
yeah, thats is good as well